src/index.ts

```diff
@@ -38,6 +39,7 @@ interface ElysiaYogaConfig<
      */
     typeDefs: TypeDefs
```

Comment:
typeDefs and resolvers should be optional because schema is given

Reply:
```
Argument of type '{ schema: GraphQLSchema; graphiql: boolean; context: ({ request }: YogaInitialContext) => Promise<{ user?: any; }>; }' is not assignable to parameter of type 'ElysiaYogaConfig<string, MaybePromise<Record<string, unknown>> | ((initialContext: YogaInitialContext) => unknown) | undefined>'.
  Type '{ schema: GraphQLSchema; graphiql: boolean; context: ({ request }: YogaInitialContext) => Promise<{ user?: any; }>; }' is missing the following properties from type 'ElysiaYogaConfig<string, MaybePromise<Record<string, unknown>> | ((initialContext: YogaInitialContext) => unknown) | undefined>': typeDefs, resolversts(2345)
  ````

Reply:
@qlaffont thanks, sorry for jumping on this so late, I have updated types, props: typeDefs, resolvers, schema are now optional 
`(property) typeDefs?: TypeSource | undefined`
`(property) resolvers?: {
    Query?: {} | undefined;
    Mutation?: {} | undefined;
    Subscription?: {} | undefined;
} | undefined`
`(property) schema?: GraphQLSchemaWithContext<MaybePromise<Record<string, unknown>> | ((initialContext: YogaInitialContext) => unknown) | undefined> | undefined`